contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -1043,7 +1043,8 @@ def get_node(self, node_id=None, options=None, dbfile=None,
         db = self.db_provider.get_db(os.path.join(lightning_dir, TEST_NETWORK), self.testname, node_id)
         node = self.node_cls(
             node_id, lightning_dir, self.bitcoind, self.executor, db=db,
-            port=port, options=options, **kwargs
+            port=port, options=options, may_fail=may_fail or expect_fail,
```

Comment:
What we usually do in existing tests is `may_fail=True, expect_fail=True` but I agree this is better and makes sense.

Reply:
Yeah, and `LightningNode()` does not have an `expect_fail` parameter either.